giskard/testing/tests/performance.py

```diff
@@ -163,25 +165,19 @@ def _test_diff_prediction(
         raise ValueError(f"Invalid direction: {direction}")
 
     # --- debug ---
-    output_ds = None
-    if not passed and debug:
-        if result_reference.output_df is None:
-            output_ds = result_actual.output_df.copy()
-        elif result_actual.output_df is None:
-            output_ds = result_reference.output_df.copy()
-        else:
-            output_ds = result_reference.output_df.copy()  # copy all properties
-            output_ds.df = pd.concat([result_actual.output_df.df, result_reference.output_df.df], ignore_index=True)
-        test_name = inspect.stack()[1][3]
-        output_ds.name = debug_prefix + test_name
+    failed_indexes = list()
+    if not passed:
+        pass
+        # TODO: How to do with dual dataset?
```

Comment:
@rabah-khalek @Inokinoki 

Not sure how to handle debugging on this one. It's a mix of two dataset (actual and reference).

My idea is to have failed_indexes has a dict: 
```python
{
  'reference_dataset': reference_dataset.index.get_indexer_for(result_reference.failed_indexes),
  'actual_dataset': actual_dataset.index.get_indexer_for(result_actual.failed_indexes),
}
```

And then have the CreateSubDataset to generate this one based on the datasets. It would be more robust but require some changes